cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,13 @@
-/// <reference types="cypress" />
-
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
 
+    cy.visit('https://conduit.mate.academy/#/login');
+
+    cy.get('input[type="email"]').type('miretta2@ukr.net');
+    cy.get('input[type="password"]').type('Miretta29+');
```

Comment:
Hardcoding the password is not a good practice for the same reasons as mentioned for the email. Please, consider moving it to a separate config file or environment variables.